Developer broggin

chauntika opened the pull request:
All issues with the notes are fixed now. It's recording one time, it edits and deletes the notes. And it also automatically refreshes after the user edits or deletes something on the page. I'm working on adding functionality to the Record button on the notes page.

chauntika committed:
more

more

chauntika committed:
Merge branch 'developer-broggin' of https://github.com/umgc/summer2021.amazing into developer-broggin

chauntika committed:
All Issues Fixed with the Notes

Everything is running. Its recording, deleting and editing the notes

jersoe reviewed (commented):
LGTM

mdrammeh1 reviewed (approved):
Approving to be able to get this out to everyone, but there looks to be a small issue with updating the page after adds/deletes. Can take a deeper look after the merge.

chauntika committed:
Automatic updating

Okay, I tested and everything should be working this time. It's working for me, but let me know if you are still having issues.

mdrammeh1 reviewed (approved):
refreshing on edits and deletes look good
